src/main/java/de/retest/web/ScreenshotProvider.java

```diff
@@ -49,7 +49,7 @@ private static BufferedImage shootViewportOnly( final WebDriver driver ) {
 	 * @return The wrapped driver if present, otherwise the given driver.
 	 */
 	private static WebDriver unwrap( final WebDriver driver ) {
-		return driver instanceof UnbreakableDriver ? ((UnbreakableDriver) driver).getWrapped() : driver;
+		return driver instanceof WrapsDriver ? ((WrapsDriver) driver).getWrappedDriver() : driver;
```

Comment:
Missing import seems to cause compilation issues: https://travis-ci.com/retest/recheck-web/builds/121609741#L537

Reply:
Done